pkg/utils/kv.go

```diff
@@ -11,7 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package cdclog
+package utils
```

Comment:
are these renames intentional? 🤔

Reply:
yes, not only br cdclog will use `kv.go`, lightning still need use it. so I put it in `utils` pkg.

Reply:
How about putting it into a new package? I don't want to package utils to be too bloat.

Reply:
OK, I put it into `kv` pkg, because these code are all related to kv.